### PR TITLE
ember-new-plus (for The Ember Times  -  Issue No. 153)

### PR DESCRIPTION
**Pull request by atej**
PR for the `ember-new-plus` section in The Ember Times - Issue No. 153

### CODE REVIEW

File: source/2020-06-19-the-ember-times-issue-153.md [hunk 1]
```diff
@@ -14,14 +14,13 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Set up fast with `ember-new-plus` ⚡️](https://github.com/atej/ember-new-plus)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Aseem Taneja (@atej)](https://github.com/atej) has created a bash script that automates the setup of some often-used tools in an ember project. To further enhance your DX, it also recommends [VS Code](https://code.visualstudio.com/) extensions and generates settings to integrate with them.
```
Comment (ijlee2) [inline source/2020-06-19-the-ember-times-issue-153.md]:
```suggestion
[Aseem Taneja (@atej)](https://github.com/atej) has created a bash script that automates the setup of some often-used tools in an Ember project. To further enhance your DX, it also recommends [VS Code](https://code.visualstudio.com/) extensions and generates settings to integrate with them.
```

File: source/2020-06-19-the-ember-times-issue-153.md [hunk 2]
```diff
@@ -14,14 +14,13 @@ READMORE
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Based on your selections, you get – linting and formatting with [ESLint](https://eslint.org) + [Prettier](https://prettier.io), utility-first CSS with [Tailwind CSS](https://tailwindcss.com) + [PostCSS](https://postcss.org), git hooks with [Husky](https://github.com/typicode/husky), pre-commit linting of staged files with [lint-staged](https://github.com/okonet/lint-staged), adherance to [conventional commit messages](https://conventionalcommits.org) with [commitlint](https://github.com/conventional-changelog/commitlint), and a launch configuration for in-editor debugging with VS Code's [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome).
```
Comment (ijlee2) [inline source/2020-06-19-the-ember-times-issue-153.md]: I thought that the sentence that you have reads okay. I think we can make your script's features more clear by making a list. What do you think?
```suggestion
Based on your selections, you get:

- Linting and formatting with [ESLint](https://eslint.org) + [Prettier](https://prettier.io),
- Utility-first CSS with [Tailwind CSS](https://tailwindcss.com) + [PostCSS](https://postcss.org)
- git hooks with [Husky](https://github.com/typicode/husky)
- Pre-commit linting of staged files with [lint-staged](https://github.com/okonet/lint-staged)
- Adherance to [conventional commit messages](https://conventionalcommits.org) with [commitlint](https://github.com/conventional-changelog/commitlint)
- A launch configuration for in-editor debugging with VS Code's [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
```

File: source/2020-06-19-the-ember-times-issue-153.md [hunk 3]
```diff
@@ -14,14 +14,13 @@ READMORE
...
+
+The configuration files generated when you first [run the script](https://github.com/atej/ember-new-plus#quickstart-) can be tuned to your preference later.
```
Comment (ijlee2) [inline source/2020-06-19-the-ember-times-issue-153.md]: I noticed that we currently only have 1 direct link to your GitHub repo (the title, which can be missed by readers). I think it'd be good to provide the link once more at the end.

Something like:
```suggestion
The configuration files generated when you first [run the script](https://github.com/atej/ember-new-plus#quickstart-) can be tuned to your preference later. If you are about to make a new Ember project, give [ember-new-plus](https://github.com/atej/ember-new-plus) a try!
```
?

**Review by ijlee2 — APPROVED** [review]
Aseem, thanks so much for sharing your news with us (and your script to set up many DX-related tools for Ember apps)!

I left a few suggestions that I think will improve the writing. I'll go ahead and approve this PR so that you can make changes and merge your PR at any time.

**Comment by ijlee2** [comment]
I pushed a commit to fix merge conflict with Matthew's PR.

**Comment by atej** [comment]
Thanks, Isaac, for taking the time to suggest the changes. I've incorporated them in the PR.

**Pull request merged**